src/OpenTracing/SpanOptions.php

```diff
@@ -43,7 +43,15 @@ public static function create(array $options)
 
                 case 'tags':
                     foreach ($value as $tag => $tagValue) {
-                        $spanOptions->tags[] = Tag::create($tag, $tagValue);
+                        if ($tag !== (string) $tag) {
+                            throw InvalidSpanOption::invalidTag($tag);
+                        }
+
+                        if (!is_scalar($tagValue) && method_exists($tagValue, '__toString')) {
```

Comment:
Can you please elaborate more about this situation? like, according to the [spec tags are scalars](https://github.com/opentracing/specification/blob/master/specification.md#set-a-span-tag) is the idea behind it is to pass more than one value for one single tag?